### PR TITLE
Render as HTML by default

### PR DESCRIPTION
**Pull request by dennisschagt**
Fixes https://github.com/newsboat/newsboat/issues/1590

While looking into that issue, I noticed that `<` is not escaped for STFL when rendering using `render_plaintext()`.

### CODE REVIEW

File: src/itemrenderer.cpp
Comment (Minoru) [inline src/itemrenderer.cpp:148]: I think `to_plain_text()` should pass a `true` for this new parameter.

File: test/itemrenderer.cpp
Comment (Minoru) [inline test/itemrenderer.cpp:729]: By the way, have you heard of "boolean blindness"? Newsboat has much, much more important issues, so I rarely think about this stuff, but it'd probably be nicer to use enums in place of bools, even when there are only two options. It'd be more readable even without creating local constants like this one (which I believe we don't always bother to do).

File: src/itemrenderer.cpp
Comment (Minoru) [inline src/itemrenderer.cpp:16]: That's *very* clever! And also way better than changing the default for the DB column to text/html, because this approach lets us distinguish between "don't know" and "it's definitely HTML" cases. Thank you very much for coming up with this solution!

**Review by Minoru — CHANGES_REQUESTED** [review]
I think you forgot to pass `false` for `raw` in one place, but other than that, this looks excellent :+1: You can merge this after adding the missing parameter.

File: src/itemrenderer.cpp
Comment (dennisschagt) [reply src/itemrenderer.cpp]: Good catch, fixed in https://github.com/newsboat/newsboat/pull/1595/commits/799baac5d8390fff5b129c138b1c80b1d562ee60

File: test/itemrenderer.cpp
Comment (dennisschagt) [reply test/itemrenderer.cpp]: Never heard of it before.

I'm now using an enum, is this what you had in mind? (https://github.com/newsboat/newsboat/commit/799baac5d8390fff5b129c138b1c80b1d562ee60)

Comment (Minoru) [reply test/itemrenderer.cpp]: Yes, exactly! I didn't mean it as a request for change, just wanted to know what you think of this approach. Should've labeled it "discussion" or something. So: do you think this approach makes more sense than `bool`s? (I do, but I'm always enthusiastic about type safety stuff even when it makes very little difference, so I'd like a second opinion.)

**Review by Minoru — APPROVED** [review]
:+1: Cool! Merging, but would still like to discuss boolean blindness.

File: test/itemrenderer.cpp
Comment (dennisschagt) [reply test/itemrenderer.cpp]: I don't really have an opinion on this but I guess it does help readability in some cases (including this place).

**Pull request merged**
